app/models/waste_carriers_engine/order_item_log.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
```

Comment:
We have this in the back office PR. Should really be in one place or the other. It's scope to `WasteCarriersEngine` so the engine would make most sense

Reply:
Agreed, this can stay in the engine and I will remove it from the back-office.